modules/apps/object/object-rest-test/src/testFunctional/macros/CustomObjectAPI.macro

```diff
@@ -203,6 +169,65 @@ definition {
 		return ${response};
 	}
 
+	macro nObjectEntryBodyAsNestedField {
+		Variables.assertDefined(parameterList = "${body},${nestedField},${numberOfRelatedObjectEntries},${relatedEntryFieldName}");
+
+		var size = ${numberOfRelatedObjectEntries};
+
+		if (!(isSet(externalReferenceCode))) {
+			var externalReferenceCode = "";
+		}
+
+		if (isSet(oneToManyChild) && (${oneToManyChild} == "true")) {
```

Comment:
The only thing related to the codes for me is I think here set ${oneToManyChild} == "true" in the condition seems enough?

Reply:
you mean to get rid of (isSet(oneToManyChild)?

Reply:
yes :small_airplane:

Reply:
yeah, I though about it is just looked so complete with the other one 😆 Will remove it now cause yeah it is unnecessary :D